app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
```

Comment:
Code Style: It would be better to raise an exception or print an error message if the command is not valid, instead of silently returning.